### PR TITLE
DataFrame.round()

### PR DESCRIPTION
**Pull request by RainFung**

### CODE REVIEW

**Comment by RainFung** [comment]
wait to code review

File: databricks/koalas/series.py [hunk 1]
```diff
@@ -2201,6 +2201,45 @@ def transform(self, func, *args, **kwargs):
...
+        >>> df.round(2)
+        0    0.03
+        1    0.04
+        2    0.88
+        Name: x, dtype: float64
+        """
+        if isinstance(decimals, int):
+            return _col(self.to_dataframe().round(decimals=decimals))
+        else:
+            raise ValueError("decimals must be an integer")
```
Comment (HyukjinKwon) [inline databricks/koalas/series.py]: @RainFung, can we do `F.round` here? Once new DataFrame is created, it causes an issue such as https://github.com/databricks/koalas/issues/405. We can avoid this via not creating new DataFrame via `to_dataframe()~.

**Comment by HyukjinKwon** [comment]
Thanks for this PR. @RainFung. Otherwise, looks pretty good to me.

File: databricks/koalas/series.py (hunk 1 above)
Comment (RainFung) [reply databricks/koalas/series.py]: Is there a need update functions which contains to_dataframe() ?

File: databricks/koalas/series.py [hunk 2]
```diff
@@ -2271,6 +2311,7 @@ def rank(self, method='average', ascending=True):
         2  2.0  2.0
         3  3.0  1.0
         """
+
```
Comment (ueshin) [inline databricks/koalas/series.py]: nit: unrelated change?

File: databricks/koalas/tests/test_dataframe.py [hunk 3]
```diff
@@ -1040,3 +1040,21 @@ def test_rank(self):
...
+    def test_round(self):
+        pdf = pd.DataFrame({'A': [0.028208, 0.038683, 0.877076],
+                            'B': [0.992815, 0.645646, 0.149370],
+                            'C': [0.173891, 0.577595, 0.491027]},
+                           columns=['A', 'B', 'C'], index=['first', 'second', 'third'])
+        kdf = ks.from_pandas(pdf)
+        pser = pd.Series([1, 0, 2], index=['A', 'B', 'C'])
+        kser = ks.Series([1, 0, 2], index=['A', 'B', 'C'])
+        self.assert_eq(pdf.round(2),
+                       kdf.round(2).sort_index())
```
Comment (ueshin) [inline databricks/koalas/tests/test_dataframe.py]: Do we need `sort_index()` here?

File: databricks/koalas/tests/test_series.py [hunk 4]
```diff
@@ -509,3 +509,12 @@ def test_rank(self):
         msg = "method must be one of 'average', 'min', 'max', 'first', 'dense'"
         with self.assertRaisesRegex(ValueError, msg):
             kser.rank(method='nothing')
+
+    def test_round(self):
+        pser = pd.Series([0.028208, 0.038683, 0.877076], name='x')
+        kser = koalas.from_pandas(pser)
+        self.assert_eq(pser.round(2),
+                       kser.round(2).sort_index())
```
Comment (ueshin) [inline databricks/koalas/tests/test_series.py]: ditto.

**Comment by RainFung** [comment]
fixed

**Comment by HyukjinKwon** [comment]
Thanks, @RainFung
